package.json

```diff
@@ -28,7 +28,6 @@
         "postbuild": "npm run copy-assets",
         "precopy-assets": "npm run docs",
         "copy-assets": "node scripts/guide-copy-assets.js",
-        "precommit": "lint-staged",
```

Comment:
I am good with this, we already have linting as a part of the pr validation. If we are removing this, we should remove the "lint-staged" section of the package.json

Reply:
Cool, good catch - just pushed a commit that removes that lint-staged section.